application/controllers/admin/PluginManagerController.php

```diff
@@ -329,7 +329,15 @@ public function configure($id)
             $aSettings = $oPluginObject->getPluginSettings(false);
             $aSave     = array();
             foreach (array_keys($aSettings) as $name) {
-                $aSave[$name] = App()->request->getPost($name, null);
+                $newValue = App()->request->getPost($name, null);
+                if (is_array($newValue) && !empty($newValue['datetime']) && !empty($newValue['datetimesaveformat'])) {
```

Comment:
Same comments that on [here](https://github.com/LimeSurvey/LimeSurvey/pull/2753/files#r1037294220)